app-modules/ai/src/Notifications/SendAssistantTranscriptNotification.php

```diff
@@ -71,7 +72,7 @@ public function toEmail(object $notifiable): MailMessage
                 ->line("Here is a copy of {$this->sender->name}'s chat with {$this->thread->assistant->name}:");
         }
 
-        $this->thread->messages()->with('user')->get()
+        $this->thread->messages()->when(SmartPromptsFeature::active(), fn ($query) => $query->whereNull('prompt_id'))->with('user')->get()
```

Comment:
Rather than filtering out the messages with a `prompt_id` I think we should probably include them but make the displayed content the same as in the UI

Reply:
Done